shaders/src/brdf.fs

```diff
@@ -78,17 +78,18 @@ float D_GGX(float roughness, float NoH, const vec3 h) {
     return saturateMediump(d);
 }
 
+float pow2( float x ) {
+    return x*x;
+}
+
```

Comment:
Are the changes in this file worth it? The way I see it is only comments/refactoring. I'd revert this

Reply:
There's the addition of Ward anisotropic distribution, which was an artist request, and used in metallic materials. The `pow2` function is not strictly necessary, it could be removed.

Reply:
While these diffs really are just cosmetic changes, I realized that this is not Google's code, but ours (introduced in #9). So I am fine with keeping the changes in this file